org.eclipse.xtext.logging/pom.xml

```diff
@@ -9,13 +9,6 @@
 	</parent>
 	<packaging>eclipse-plugin</packaging>
 	<artifactId>org.eclipse.xtext.logging</artifactId>
-	<version>1.2.24-SNAPSHOT</version>
```

Comment:
@szarnekow I think that's required, otherwise, the Maven version will not match with the MANIFEST version `1.2.24.qualifier`, and since this is a fragment, it should have the same version of the log4j bundle.

Reply:
@szarnekow as I said in the general comments, it used to be like that in the old days before repo merging and it had to be updated consistently with the version of log4j: @cdietrich told me that a fragment must have the same number of the hosting bundle; maybe it was something we didn't know about or it was like that in the past. I'm happy we can get rid of that inconsistent version :)